src/pyOpenMS/pxds/ChromatogramTools.pxd

```diff
@@ -9,10 +9,10 @@ cdef extern from "<OpenMS/KERNEL/ChromatogramTools.h>" namespace "OpenMS":
 
         void convertChromatogramsToSpectra(
                 MSExperiment & epx
-                ) nogil except +
+                ) nogil except + # wrap-doc:Converts the chromatogram to a list of spectra with instrument settings SRM
```

Comment:
Note to us: check the SRM in the end. Does it make sense? Should probably be improved.